src/main/java/LacksDataBaseProject/User.java

```diff
@@ -0,0 +1,84 @@
+package LacksDataBaseProject;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class User {
+    private String userName;
+    private String login;
+    private String password;
+    private String skypeID;
+    private Role role;
+
+    static List<User>listOfUsers = new ArrayList<>();
+
+    public User() {
+    }
+
+    public User(String userName, String login, String password, String skypeID, Role role) {
+        this.userName = userName;
+        this.login = login;
+        this.password = password;
+        this.skypeID = skypeID;
+        this.role = role;
+    }
+
```

Comment:
Ja sobie zostawiłem pusty konstruktor, bo to ułatwia testowanie w main'ie - nie musze ciagle wpisywac tych bzdur zeby stworzy obiekt.
Gettery i settery moglyby byc tylko te ktore sa uzywane, bo niepotrzebne te nadmiarowe linijki kodu - i tak bym robil wszedzie.